src/EnergyPlus/AirLoopHVACDOAS.hh

```diff
@@ -184,23 +185,23 @@ namespace AirLoopHVACDOAS {
         {
         }
 
-        static void getAirLoopDOASInput();
+        static void getAirLoopDOASInput(EnergyPlusData &state);
 
-        void SimAirLoopHVACDOAS(bool const firstHVACIteration, int &CompIndex);
+        void SimAirLoopHVACDOAS(EnergyPlusData &state, bool const firstHVACIteration, int &CompIndex);
 
-        void initAirLoopDOAS(bool const FirstHVACIteration);
+        void initAirLoopDOAS(EnergyPlusData &state, bool const FirstHVACIteration);
 
-        void CalcAirLoopDOAS(bool const FirstHVACIteration);
+        void CalcAirLoopDOAS(EnergyPlusData &state, bool const FirstHVACIteration);
 
-        void SizingAirLoopDOAS();
+        void SizingAirLoopDOAS(EnergyPlusData &state);
```

Comment:
Add EnergyPlusData &state to all methods that need the formerly global variables.